pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -129,4 +129,6 @@ const (
 	BuildStrategyJenkinsPipelineRoleBindingName = BuildStrategyJenkinsPipelineRoleName + "-binding"
 
 	OpenshiftSharedResourceViewRoleBindingName = OpenshiftSharedResourceViewRoleName + "s"
+
+	TemplateInstanceControllerRoleBindingName = TemplateInstanceControllerRoleName
```

Comment:
@deads2k previously requested `+ "-binding"` to be removed in order to start matching upstream.